cli/tests/testdata/jsr/registry/@denotest/subset_type_graph/0.1.0/mod.ts

```diff
@@ -0,0 +1,16 @@
+// add some statements that will be removed by the subset
+// type graph so that we can test that the source map works
+console.log(1);
+console.log(2);
+console.log(3);
+
+export class Foo {
+  method(): number {
+    return Math.random();
+  }
+}
+
+// this won't be type checked against because the subset
+// type graph will ignore it
+const invalidTypeCheck: number = "";
+console.log(invalidTypeCheck);
```

Comment:
Could describe the reason for different behavior between this...

Reply:
I updated the comments for these. Let me know if the new comments don't explain it well enough.

Reply:
That's good, thanks!